kubernetes/samples/scripts/create-weblogic-domain/model-in-image/build-model-image.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+# 
+#  Summary:
+#
+#    This script builds a model image using the WebLogic Image Tool. The
+#    tool pulls a base image if there isn't already a local base image.
+#    This script, by default, builds the model image with model files from
+#    './stage-model-image.sh' plus tooling downloaded by './stage-tooling.sh.'.
+#
+#  Assumptions:
+#
+#    - The WebLogic Image Tool zip is 'WORKDIR/weblogic-image-tool.zip' and
+#      the WebLogic Deploy Tool zip is 'WORKDIR/weblogic-deploy-tooling.zip'
+#      (see './stage-tooling.sh').
+#
+#    - Model files have been staged in the MODEL_DIR directory
+#      (by './stage-model-image.sh' or some custom process).
+#
+#  Optional environment variables:
+#
+#    WORKDIR
+#      Working directory for the sample with at least 10GB of space.
+#      Defaults to '/tmp/$USER/model-in-image-sample-work-dir'.
+#
+#    MODEL_DIR:
+#      Location of the model .zip, .properties, and .yaml files
+#      that will be copied to the model image.  Default is:
+#        'WORKDIR/model/image--$(basename $MODEL_IMAGE_NAME):$MODEL_IMAGE_TAG'
+#      which is usually populated by the './stage-model-image.sh' script.
+#
+#    MODEL_IMAGE_BUILD:
+#      Set to 'when-changed' (default) or 'always'. Default behavior is to
```

Comment:
This doesn't belong in this tool, but outside of it.

Reply:
What doesn't?

Reply:
The default behavior to skip rebuild when image already exists was insisted on by Mark.

Reply:
And I agree that it's helpful...